tests/convert_numbers_test.c

```diff
@@ -28,8 +28,18 @@
 #include <inttypes.h>
 #include <float.h>
 
-static const int64_t ints[]={12, -114 , 1 , 134 , 43243 , 0, -215025, INT64_MIN, INT64_MAX};
-static const char* ints_text[] = {"12","-114","1", "134", "43243","0","-215025", "-9223372036854775808", "9223372036854775807"};
+static const int64_t ints[] = {12, -114, 1, 134, 43243, 0, -215025, -4294967297, 4294967297, INT64_MIN, INT64_MAX};
```

Comment:
Why exactly -4294967297 and 4294967297? Would rather have expected something around INT32_MAX/INT32_MIN (for the value as well as for readability)

Reply:
Important to test are values which overflow int32 and are not exactly `INT64_MIN`, `INT64_MAX`. Looked for better suiting numbers which are more self-explanatory.

Reply:
How about `INT32_MAX + 1` and `INT32_MIN - 1`? Maybe with some casting if need be?